pkgs/applications/networking/mailreaders/notmuch/default.nix

```diff
@@ -40,6 +47,19 @@ stdenv.mkDerivation rec {
   ] ++ lib.optional withRuby ruby
     ++ lib.optional withSfsexp sfsexp;
 
+  patches = [
+    (fetchpatch {
```

Comment:
Too bad upstream uses mailing lists for tracking development... I know its a lot to ask, but are you aware of any attempts to tell them about these issues? It'd be nice to add some comment just in case.